test/node.js

```diff
@@ -25,6 +41,16 @@ describe('IPFS Repo Tests onNode.js', () => {
       lock: 'memory'
     },
     init: true
+  }, {
+    name: 'custom locker',
```

Comment:
There is no proper locking tests currently, so it would be good to test various locking scenarios in addition to the regular tests